client/landing/gutenboarding/onboarding-block/plans/plan-item.tsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useI18n } from '@automattic/react-i18n';
+import { Button, Icon } from '@wordpress/components';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+
+const TickIcon = () => (
+	<Icon
+		icon={ () => (
```

Comment:
Since this needs no props => doesn't need to ever rerender. I don't think it needs to be a function. 

```suggestion
const TickIcon = (
	<Icon
		icon={ () => (
```

Then use it as `{TickIcon}`